src/rgw/rgw_main.cc

```diff
@@ -591,6 +593,15 @@ static int process_request(RGWRados *store, RGWREST *rest, RGWRequest *req, RGWC
     abort_early(s, op, -ERR_USER_SUSPENDED);
     goto done;
   }
+
+  if ( s->user.user_id != RGW_USER_ANON_ID ) {
+    if ( rgw_rate_limit_ok(s->user.user_id, op) == false ) {
+      dout(20) << "user: " << s->user.user_id << "exceeded API limit" << dendl;
```

Comment:
I intended this to be used only during testing or similar, else wouldn't processing be slowed down due to excessive(?) logging ?